lib/mimic/module.ex

```diff
@@ -77,13 +77,18 @@ defmodule Mimic.Module do
     [:return_errors | [:debug_info | options]]
   end
 
-  defp load_binary(module, binary) do
+  defp load_binary(module, binary, cover_enabled?) do
     case :code.load_binary(module, '', binary) do
       {:module, ^module} -> :ok
       {:error, reason} -> exit({:error_loading_module, module, reason})
     end
 
-    apply(:cover, :compile_beams, [[{module, binary}]])
+    if cover_enabled? do
+      Cover.export_private_functions()
```

Comment:
Felt that this was the better place to put the export. So the export of private functions is right next to the usage of the private function

Reply:
Yup that's a good place :+1: